fmriprep/workflows/bold/base.py

```diff
@@ -211,7 +211,8 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         * :py:func:`~fmriprep.workflows.bold.stc.init_bold_stc_wf`
         * :py:func:`~fmriprep.workflows.bold.hmc.init_bold_hmc_wf`
         * :py:func:`~fmriprep.workflows.bold.t2s.init_bold_t2s_wf`
-        * :py:func:`~fmriprep.workflows.bold.registration.init_bold_reg_wf`
+        * :py:func:`~fmriprep.workflows.bold.registration.init_bold_apply_reg_wf`
+        * :py:func:`~fmriprep.workflows.bold.registration.init_bold_calc_reg_wf`
```

Comment:
For consistency, let's call the apply registration workflow and `init_bold_t1_trans_wf`. We could then go back to `init_bold_reg_wf` or `init_bold_calc_reg_wf`; either one.